blockchain/contracts/NFTtheWorld.sol

```diff
@@ -3,23 +3,31 @@
 
 pragma solidity ^0.8.0;
 
+//import "./AddrArrayLib.sol";
+
+
 contract NFTtheWorld {
     address payable nftOwner;
 
+    address payable[] adminAddresses;
+
     uint256 percentageLimit;
 
     address public user;
 
     uint256[] public dropHashes;
 
     struct NFTOwnership {
-        address owner;
+        address payable owner;
         uint256 nftId;
         uint256 dropId;
         uint256 dropTime;
+        uint256 weiPrice;
         address reservedFor;
     }
 
+    mapping(address => bool) public adminRights;
```

Comment:
Does this need to be public? I don't think it destroys anything, just curious.

Reply:
Hmm no not in general, but for right now & testing purposes it suits good I think